### DISCUSSION TITLE
Using MvT programatically

### DISCUSSION THREAD

**Discussion by AdamC03** [mvt-project/mvt#362] [Q&A]
I would like to run MvT functions from within a python script. Currently I am running these by running a `os.system("MvT command")` call from within a python script. This is annoying as I would ideally like to have a wider range of controls and be able to run individual MvT functions via an import into the script. For example:
`import mvt
mvt.decrypt("backup path")
`

I know this decrypt function may not be a real function that works like this within mvt but this is just an example on how I would see it working. Is there any way I can import and run functions from the library without needing to run them via the cli/os.system?

**Comment by Te-k** [comment]
Hi, yes you can but you need to understand a bit how MVT code works. For backup decryption for instance, you can use the class `decryptbackup` in [mvt/ios/decrypt.py](https://github.com/mvt-project/mvt/blob/main/mvt/ios/decrypt.py#L21). For other features, I encourage you to look at how the CLI is implementing them, like [here for iOS](https://github.com/mvt-project/mvt/blob/main/mvt/ios/cli.py).
Last thing : these functions may evolve so I strongly suggest that you fix the MVT version in your code requirements.
